src/crawler/scraper.py

```diff
@@ -51,17 +50,15 @@ def count_reviews_each_month(self):
             return
         dir_name = self.name_of_app
         file_name = self.__get_file_name(1)
-        path = ".\\" + dir_name + "\\" + file_name
+        path = './/' + dir_name + '//' + file_name
         df = pd.read_csv(path)
         for score in range(2, 6):
             file_name = self.__get_file_name(score)
-            path = ".\\" + dir_name + "\\" + file_name
+            path = './/' + dir_name + '//' + file_name
             df1 = pd.read_csv(path)
             df = df.append(df1)
```

Comment:
This code is a duplicate in count_reviews_each_week and  count_reviews_each_month. Could you merge move this code to a private function?.

Reply:
Done